crates/autopilot/src/database/onchain_order_events.rs

```diff
@@ -655,6 +669,28 @@ fn extract_order_data_from_onchain_order_placement_event(
     Ok((order_data, owner, signing_scheme, order_uid))
 }
 
+#[derive(prometheus_metric_storage::MetricStorage, Clone, Debug)]
+#[metric(subsystem = "onchain_orders")]
+struct Metrics {
+    /// Keeps track of errors in picking up onchain orders.
+    /// Note that an order might be created even if an error is encountered.
+    #[metric(labels("error_type"))]
+    onchain_order_errors: prometheus::IntCounterVec,
```

Comment:
Does it make sense to initialize all counters with 0 on startup so that you don't have missing values?

Reply:
I think it is fine to leave as is and revisit if ever this causes issues.